apps/ledger-live-desktop/src/renderer/families/bitcoin/AccountHeaderManageActions.ts

```diff
@@ -7,13 +7,18 @@ import { stakeDefaultTrack } from "~/renderer/screens/stake/constants";
 import { BitcoinAccount } from "@ledgerhq/coin-bitcoin/lib/types";
 import { TokenAccount } from "@ledgerhq/types-live";
 import IconCoins from "~/renderer/icons/Coins";
+import useFeature from "@ledgerhq/live-common/featureFlags/useFeature";
 
 type Props = {
   account: BitcoinAccount | TokenAccount;
   parentAccount: BitcoinAccount | undefined | null;
 };
 
 const AccountHeaderActions = ({ account, parentAccount }: Props) => {
+  const stakeProgramsFeatureFlag = useFeature("stakePrograms");
```

Comment:
You can do something like that too to directly have enabled and params
```  
const { params, enabled: stakeProgramsEnabled } = useFeature("stakePrograms") || {};
```

Reply:
again i took this from the rest of the codebase, but let me know if you want me to change it